pkg/kubeclient/client_test.go

```diff
@@ -0,0 +1,13 @@
+package kubeclient
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
I needed to fetch this manually using `GO111MODULE=off go get -v k8s.io/client-go/kubernetes/fake`. I think this needs to be added to the dependency list in the `Makefile`.